docs/api.rst

```diff
@@ -18,4 +18,14 @@ Roasting
    :toctree: _autosummary
 
    prommis.roasting.ree_feed_roaster
-   prommis.roasting.ree_oxalate_roaster
\ No newline at end of file
+   prommis.roasting.ree_oxalate_roaster
+
+Precipitate
+-----------
+
+.. autosummary::
+   :toctree: _autosummary
+   
+   prommis.precipitate.precipitator
+   prommis.precipitate.precipitate_liquid_properties
+   prommis.precipitate.precipitate_solid_properties
```

Comment:
I think there's a mismatch between module names that's causing the current error: the module is named `precipitate_solids_properties` while the entry here is `precipitate_solid_properties`.